profil.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Quiz-App</title>
+    <link rel="stylesheet" href="./css/styles.css" />
+  </head>
+  <body>
+    <header class="header">
+      <h1>Quiz-App</h1>
+    </header>
+    <main>
+        <section class="imgname">
+            <figure>
+                <img
+             class="pflanze"
```

Comment:
i think "userpicture" or "profileimg" would fit better, because "pflanze" is only a placeholder here